formal-spec/decentralized-updates.tex

```diff
@@ -50,6 +51,28 @@
 
 \newcommand{\nnote}[1]{{\color{blue}\small Nikos: #1}}
 
+
+\lstdefinestyle{mystyle}{
+%    backgroundcolor=\color{backcolour},   
```

Comment:
Maybe remove things that are commented out?

Reply:
I thought that maybe we can leave the comments that correspond to a few specific key-words/attributes of the coding-list style definition, so as not to have to look for them, in the case we need them.